lv_conf.h

```diff
@@ -9,14 +9,15 @@
 /* clang-format off */
 
 #include <stdint.h>
+#define mp_instance_cast_to_native_base mp_obj_cast_to_native_base
```

Comment:
I don't see this being used anywhere else in the diff; is there a reason why it's here?

Reply:
Yes. It is needed to compile with latest micropython where *instance* was replaced with *obj* 
To avoid cross editors issues I formatted all examples with tools/codeformat.py

Reply:
@amirgon Should `gen_mpy.py` be updated?

Reply:
Honestly I don't know how! May you help me?

Reply:
@embeddedt `gen_mpy.py` is aligned to last official Micropython version.
I think @prolomb wants the code to be compatible with a more recent (unofficial) version, so this define is a workaround.  
After a new Micropython version is out, I'll be able to fix `gen_mpy.py` (and some other drivers)  and remove this define.
For now, I don't think any change is needed on `gen_mpy.py`.

Reply:
Okay; that's fine.